app/components/ui/Navigation.tsx

```diff
@@ -41,29 +41,40 @@ export const Navigation = ({
 
         <ul className={styles.navRight}>
           {menus?.map((menu) => (
-            <div className={styles.menuContainer} key={menu.id.toString()}>
+            <div
+              className={styles.navigationMenuContainer}
+              key={menu.id.toString()}
+            >
               <button
                 type="button"
                 aria-haspopup="menu"
-                aria-expanded={dropdown ? "true" : "false"}
+                aria-expanded={dropdownIsOpen ? "true" : "false"}
                 onClick={() => setDropdown((prev) => !prev)}
+                className={styles.navigationMenuHeader}
               >
                 {menu.title}
+                <span className={`fas fa-chevron-down ${styles.chevron}`} />
               </button>
 
               <ul
-                className={`${styles.dropdown} ${
-                  dropdown ? styles.showDropdown : ""
+                className={`${styles.dropdown} ${styles.navigationMenuList} ${
+                  dropdownIsOpen ? styles.showDropdown : ""
                 }`}
               >
                 {menu.link.map((linkItem: StrapiModel.Link) => (
                   <li key={linkItem.id} className="menu-item">
-                    <Link to={linkItem.url}>{linkItem.text}</Link>
+                    <Link
+                      to={linkItem.url}
+                      className={styles.navigationMenuLink}
+                    >
+                      {linkItem.text}
+                    </Link>
                   </li>
                 ))}
               </ul>
             </div>
           ))}
+          <div>|</div>
```

Comment:
Since this is text and has no meaning to the user, it will likely cause a11y problems. Can you add a border-left or border-right to one of these elements in the nav instead?

Reply:
Good call!